python/pyspark/ml/fpm.py

```diff
@@ -186,29 +186,29 @@ class FPGrowth(JavaEstimator, HasItemsCol, HasPredictionCol,
     |[z]                     |
     |[x, z, y, r, q, t, p]   |
     +------------------------+
-    >>> fp = FPGrowth(minSupport=0.2, minConfidence=0.7)
+    >>> fp = FPGrowth(minSupport=0.4, minConfidence=0.7)
     >>> fpm = fp.fit(data)
     >>> fpm.freqItemsets.show(5)
-    +---------+----+
-    |    items|freq|
-    +---------+----+
-    |      [s]|   3|
-    |   [s, x]|   3|
-    |[s, x, z]|   2|
-    |   [s, z]|   2|
-    |      [r]|   3|
-    +---------+----+
+    +------+----+
+    | items|freq|
+    +------+----+
+    |   [s]|   3|
+    |[s, x]|   3|
+    |   [r]|   3|
+    |   [y]|   3|
+    |[y, x]|   3|
+    +------+----+
```

Comment:
or rather, why is https://github.com/apache/spark/pull/17280/files#diff-b6dbf16870bd2cca9b4140df8aebd681L189 changed?